compiler/daml-lf-proto/BUILD.bazel

```diff
@@ -14,10 +14,12 @@ da_haskell_library(
         "either",
         "lens",
         "memory",
+        "mtl",
```

Comment:
Can you drop the mtl dependency? Generally everything is in transformers anyway, just at a different module name.

Reply:
I use `MonadError`, which is missing. I could redefine it myself...

Reply:
Is https://hackage.haskell.org/package/transformers-0.5.6.2/docs/Control-Monad-Trans-Except.html not good enough? (I'm not against an import of mtl, but usually its better to be consistent, and usually its better to use transformers)

Reply:
I don't use it for `ExceptT`, I use it so `throwError` is polymorphic to whether or not I've stacked `ReaderT` on.